From 565f72bedacda9667b5094de11bcb2659589d5a6 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 22 Jul 2014 14:38:48 -0400
Subject: [virt] kvm: make KVM_MMU_AUDIT help text more readable

Message-id: <1406040016-3289-134-git-send-email-pbonzini@redhat.com>
Patchwork-id: 86212
O-Subject: [RHEL7 PATCH v2 133/221] kvm: make KVM_MMU_AUDIT help text more readable
Bugzilla: 1116936
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Randy Dunlap <rdunlap@infradead.org>

Make KVM_MMU_AUDIT kconfig help text readable and collapse
two spaces between words down to one space.

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Reviewed-by: Xiao Guangrong <xiaoguangrong@linux.vnet.ibm.com>
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
(cherry picked from commit 94491620e1362f6065ab821c13eb54b716ada19f)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kvm/Kconfig b/arch/x86/kvm/Kconfig
index b89c5db..287e4c8 100644
--- a/arch/x86/kvm/Kconfig
+++ b/arch/x86/kvm/Kconfig
@@ -80,7 +80,7 @@ config KVM_MMU_AUDIT
  depends on KVM && TRACEPOINTS
  ---help---
   This option adds a R/W kVM module parameter 'mmu_audit', which allows
-  audit  KVM MMU at runtime.
+  auditing of KVM MMU events at runtime.
 
 config KVM_DEVICE_ASSIGNMENT
  bool "KVM legacy PCI device assignment support"
-- 
1.7.1